data/windows.yaml

```diff
@@ -423,17 +423,36 @@ provides: [environ_allusersappdata]
 supported_os: [Windows]
 urls: ['http://environmentvariables.org/ProgramData']
 ---
+name: WindowsEnvironmentVariableProfilesDirectory
+doc: Folder that typically contains users' profile directories; default is '%SystemDrive%\Users'
+sources:
+- type: REGISTRY_VALUE
+  attributes:
+    key_value_pairs:
+      - {key: 'HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\ProfileList', value: 'ProfilesDirectory'}
+provides: [environ_profilesdirectory]
+supported_os: [Windows]
+urls:
+  - 'https://msdn.microsoft.com/en-us/library/windows/desktop/bb776892(v=vs.85).aspx'
```

Comment:
For future changes, please remove indentation of `-`, I'll merge this PR manually.